### DISCUSSION TITLE
Can Default Selected Choice Be Programatically Set?

### DISCUSSION THREAD

**Discussion by prplxr** [johnlindquist/kit#719] [Q&A]
Hi, Thanks for creating such amazing tool.
I am trying to build a tab browser for firefox and ordering of choices matter. Is there a way to select a choice in middle of choices by default? instead of always selecting first one?

i tried using `focused` attribute of choice but it seems to automatically select it and close window.

**Comment by johnlindquist** [comment]
Not currently. I'll add the idea to the list for the next build. (I'm moving next week, so it might be a couple weeks)

> Note: `focused` is like `description`, but only shows up when the item is focused.
> Double note: Looks like `focused` is broken in the current build, but it was a super quick fix and will be released soon:

https://user-images.githubusercontent.com/36073/152644093-1128f9ee-df95-4013-89a2-36799bfc2f94.mp4
